django_redis/cache.py

```diff
@@ -105,7 +103,8 @@ def delete(self, *args, **kwargs):
 
     @omit_exception
     def delete_pattern(self, *args, **kwargs):
-        kwargs["itersize"] = kwargs.get("itersize", DJANGO_REDIS_SCAN_ITERSIZE)
+        if not kwargs.get("itersize"):
```

Comment:
I think it would be cleaner to get eternize from settings in the __init__ method like you did in HerdClient

Reply:
Done! Testing become trickier because Django cache reuse the same instance for all tests, so I have to test with a fixture to destroy the instance, change setting and destroy again after test to avoid other tests using a wrong setting somewhere else.